char/liberate/FULL_MODELS/r+c.mrp1monte.spice

```diff
@@ -0,0 +1,35 @@
+* Copyright 2020 The SkyWater PDK Authors
```

Comment:
is that intentional to have sky130 models being referenced here?

Reply:
deleted (just for characterization)